packages/shared/components/tiles/ActivityTile.svelte

```diff
@@ -33,7 +33,9 @@
                 ? getTokenFromSelectedAccount(activity.assetId)
                 : undefined)
     $: isTimelocked = activity?.asyncData?.timelockDate > $time
-    $: shouldShowAsyncFooter = activity.asyncData && activity.asyncData.asyncStatus !== ActivityAsyncStatus.Claimed
+    $: shouldShowAsyncFooter =
```

Comment:
I wouldnt mix the async footer with this, the idea of the async footer is that is reserved for async transactions that need claiming or are timelocked, but a conflicting transaction, imo, does not need to display this info